app/app.controllers.js

```diff
@@ -45,8 +46,20 @@ exports.getCommentsByArticle = (req, res, next) => {
   const { article_id } = req.params;
   selectComment(article_id)
     .then((article) => {
-      res.status(200).send({article});
-    }).catch((err=>{
-      next(err)
-    }));
+      res.status(200).send({ article });
+    })
+    .catch((err) => {
+      next(err);
+    });
+};
+exports.postComment = (req, res, next) => {
+  const { body, params } = req;
+
+  const passedComment = body;
+  passedComment.article_id = params.article_id;
```

Comment:
this is puzzling.  What are you trying to do here?

Reply:
I was trying to pass the body only 1 object parameter